apps/dgst.c

```diff
@@ -491,8 +491,8 @@ static void show_digests(const OBJ_NAME *name, void *arg)
     /* Filter out message digests that we cannot use */
     md = EVP_MD_fetch(app_get0_libctx(), name->name, app_get0_propq());
     if (md == NULL) {
-        md = EVP_get_digestbyname(name->name);
-        if (md == NULL)
+        const EVP_MD *md_by_name = EVP_get_digestbyname(name->name);
+        if (md_by_name == NULL)
```

Comment:
So the style checker wants a blank line above this line; maybe it would be better without the temporary variable, given that it's a dead store:

```
    /* Filter out message digests that we cannot use */
    md = EVP_MD_fetch(app_get0_libctx(), name->name, app_get0_propq());
    if (md == NULL) {
        /* See if name->name is the name of a digest */
        if (EVP_get_digestbyname(name->name) == NULL)
            return;
    }
```

Reply:
Happy to make it that way... just waiting for the compiles to complete to make sure there's nothing else to fix.

Reply:
Just not entirely sure of the code-style preferences round here ;)